packages/headless/src/controllers/facets/range-facet/date-facet/headless-date-facet-options.ts

```diff
@@ -126,7 +160,7 @@ export const dateFacetOptionsSchema = new Schema<Required<DateFacetOptions>>({
 
 export function validateDateFacetOptions(
   engine: SearchEngine<ConfigurationSection & SearchSection & DateFacetSection>,
-  options: DateFacetOptions
+  options: RegisterDateFacetActionCreatorPayload
```

Comment:
Similar to above, shouldn't the validation be on the full facet options? or should the function name be adjusted?
If the latter, could we reuse the validation logic inside the `registerDateFacet` action?